### PR TITLE
[REQUEST] Multi-User Media Cleaner

### PR DESCRIPTION
**Pull request by Bugreporter48**
**Project Description**
Query your Jellyfin server for the movies, tv episodes, audio tracks, and audiobooks in your libraries. Then delete media items you no longer want taking up disk space.

**Project link**
https://github.com/terrelsa13/MUMC

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@
 - [wizarr](https://github.com/Wizarrrr/wizarr) - Advanced user invitation and management system.
 - [ytdl-sub](https://github.com/jmbannon/ytdl-sub) - Automate downloading and metadata generation with YoutubeDL.
 - [xsrv.jellyfin](https://github.com/nodiscc/xsrv/tree/master/roles/jellyfin) - Ansible role to deploy and configure Jellyfin.
-
+- [Multi-User Media Cleaner](https://github.com/terrelsa13/MUMC) - Query your Jellyfin server for the movies, tv episodes, audio tracks, and audiobooks in your libraries. Then delete media items you no longer want taking up disk space.
```
Comment (darmiel) [inline README.md:67]: I think the description can be a bit shorter, e. g.:

```suggestion
- [Multi-User Media Cleaner](https://github.com/terrelsa13/MUMC) - Query and delete unwanted media content from your Jellyfin server.
```

Comment (darmiel) [inline README.md:67]: Also, please make sure that the order is correct. `Multi-User Media Cleaner` should sit between `jfa-go` and `Playlifin`

Comment (Bugreporter48) [reply README.md]: Changed

Comment (Bugreporter48) [reply README.md]: Done

**Review by darmiel — APPROVED** [review]

**Review by TomRomeo — APPROVED** [review]

**Pull request merged**